relay-event-normalization/src/normalize/mod.rs

```diff
@@ -918,6 +941,48 @@ impl Default for LightNormalizationConfig<'_> {
             span_description_rules: Default::default(),
             geoip_lookup: Default::default(),
             enable_trimming: false,
+            dynamic_measurements_config: None,
+        }
+    }
+}
+
+/// Container for global and project level [`MeasurementsConfig`]. The purpose is to handle
+/// the merging logic.
+#[derive(Clone, Debug)]
+pub struct DynamicMeasurementsConfig<'a> {
```

Comment:
_nit_: What would the "Dynamic" part in the name refer to? Isn't this rather a "merged" or "combined" measurements config?

Reply:
The crate is called `relay-dynamic-config` so it seems to fit the naming convention, this is the measurements part of the dynamic config.